src/app/app.component.ts

```diff
@@ -43,7 +43,7 @@ export class AppComponent implements OnInit, AfterViewInit, OnDestroy {
   protected actionsSidenavOpen = model<boolean>(false);
   protected notificationsSidenavOpened = model<boolean>(false);
   protected isPhonePortrait: Signal<BreakpointState>;
-  protected notificationsVisibility: string = 'hidden';
+  protected notificationsVisibility = 'hidden';
```

Comment:
Why remove the variable type, does it not help the compiler to gard against type issues? Or is the value assignment on declaration sufficient for the compiler?